libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -228,23 +229,30 @@ void _usage(void)
 {
     printf("Usage: -A uartAPath -B uartBPath -C uartCPath -D uartDPath -E uartEPath -F uartFPath\n");
     printf("Options:\n");
-    printf("\t-serial:          -A /dev/ttyO4\n");
+    printf("\tserial:\n");
+    printf("                    -A /dev/ttyO4\n");
     printf("\t                  -B /dev/ttyS1\n");
-    printf("\t-tcp:             -C tcp:192.168.2.15:1243:wait\n");
+    printf("\tnetworking tcp:\n");
+    printf("\t                  -C tcp:192.168.2.15:1243:wait\n");
     printf("\t                  -A tcp:11.0.0.2:5678\n");
     printf("\t                  -A udp:11.0.0.2:14550\n");
+    printf("\tnetworking UDP:\n");
     printf("\t                  -A udp:11.0.0.255:14550:bcast\n");
     printf("\t                  -A udpin:0.0.0.0:14550\n");
-    printf("\t-custom log path:\n");
+    printf("\tcustom log path:\n");
     printf("\t                  --log-directory /var/APM/logs\n");
     printf("\t                  -l /var/APM/logs\n");
-    printf("\t-custom terrain path:\n");
+    printf("\tcustom terrain path:\n");
     printf("\t                   --terrain-directory /var/APM/terrain\n");
     printf("\t                   -t /var/APM/terrain\n");
+    printf("\tmodule support:\n");
+    printf("\t                   --module-directory /etc/APM/modules\n");
```

Comment:
I'd prefer this to be /usr/lib/ardupilot/modules.  I also think we should always look at this directory and the --module-directory would be for the user just to choose another one.

Imagine we have external packages installing modules for ardupilot. Where should they install? They won't know if we don't provide a default dir.